dev/attach_security_grp_to_port.template

```diff
@@ -0,0 +1,51 @@
+heat_template_version: 2015-04-30
+description: Test template to create port wit ip_address.
+parameters:
+  mac:
+    type: string
+    default: 00-00-00-00-BB-BB
+resources:
+  server1:
+    type: OS::Nova::Server
+    properties:
+      name: Server1
+      image: { get_param: image }
+      flavor: { get_param: flavor }
+      key_name: { get_param: key_name }
+      networks:
+        - port: { get_resource: server1_port }
```

Comment:
If I'm correct in the rest of these comments, you'll need to remove the network specification here or specifically add service and public nets.